src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTaskTest.java

```diff
@@ -150,40 +132,6 @@ public void should_process_other_zip_files_if_previous_zip_fails_to_process() th
             .uploadDocuments(ImmutableList.of(new Pdf("1111001.pdf", test1PdfBytes)));
     }
 
-    @Test
```

Comment:
Are these removed integration tests going to be replaced by other tests?

Reply:
Those where moved to `BlobProcessorTaskTestForFailedStatus` suite. Forgot to remove in the first place